### ISSUE TITLE
[BUG] Dark mode lacks contrast

### ISSUE THREAD

**Issue by YunzheZJU** [YunzheZJU/holo-schedule#58]
#### Context
> One minor criticism I have is that the current dark mode does not have enough contrast. As in, the whole UI looks like it has a transparent black overlay that makes everything darker than normal. Also, maybe an option to switch between light and dark mode.

#### TODO
* Color scheme in dark mode should be updated to adjust the brightness.
* Whether to add a manual switch on light and dark mode depends on further information.

Previous issue: #47. Further updates will go into this thread.

Possibly related to user: @louie-github. Sorry for my bothering and you are free to mute this thread.

**Comment by louie-github** [comment]
Oh, hi! That was quite kind of you to consider my request and I appreciate it.

I realize what you were going for when you made the dark mode and it does work well for the use case that you described. A dark mode with too much contrast might be jarring to the eyes, but I believe that using a dark grey color instead of black, like what you did right now, could help to mitigate that.

> However, I need more information on the suggestion for the switch between light and dark mode. Would you rather use a light theme in a dark-themed OS(or in reverse)? Would you still need it if I have successfully adjusted the contarst in dark mode?

To be honest, I would probably use the dark mode if the contrast was adjusted. However, the light mode also sometimes works well for readability and I might prefer it over time for a small pop-up like what happens with the extension contrary to a full-on web page.

I just suggested to add a switch because I assumed (correct me if I'm wrong) that both light and dark modes are implemented and can be toggled, but the logic behind choosing the theme is just done behind the scenes. In that case, I thought it would be relatively simple to give the user the freedom to choose in the form of a switch.

It also makes it more consistent with a lot of other dark mode toggles in different apps. They usually give three options: *light*, *dark*, and *system default*. *System default* usually refers to following the system dark mode setting. I think this option is close to the current behavior of the extension which is to check for the theme of the browser and change accordingly.

In short, adding a theme switcher would allow the user to pick their personal preference. Once again, thank you for your time! I wish I could help out more.

**Comment by YunzheZJU** [comment]
I have updated the colors and brightness in dark mode. You can get a preview in #59. 

Beriefly, Black is lighter and White is brighter but the overall contrast is preserved (and gains a little boost). It will be left to the users to adjust the screen brightness if they feel it jarrying and all HoloSchedule should do is to match its colors/brightness with the browser's theme.

I hope you would like it! Feel free to throw any suggestions here.

**Comment by YunzheZJU** [comment]
I'm also implementing the switch for different color modes #62, but I'm not sure whether 'Appearance' is a proper name for this option.

**Comment by YunzheZJU** [comment]
I'm now publishing the latest version which contains relevant changes. It may take some time for the store side to finish the audit.

I sincerely appreciate it for your participation in improving the quality of this project and am looking forword to your further feedbacks!
